src/colorizer/recorders/Mp4VideoRecorder.ts

```diff
@@ -0,0 +1,150 @@
+import { ArrayBufferTarget, Muxer } from "mp4-muxer";
+import CanvasRecorder, { RecordingOptions, defaultRecordingOptions } from "./CanvasRecorder";
+import { sleep } from "../utils/timing_utils";
+
+// Eslint doesn't recognize the WebCodecs API yet
+/* global VideoEncoder, VideoFrame, VideoEncoderConfig*/
```

Comment:
is this still meant to be here?

Reply:
Yes, updated the comment to mention the specific linter errors that get thrown without it 😞